### ISSUE TITLE
Non-profit Member should have Link to Share holder 

### ISSUE THREAD

**Issue by kothagunda** [frappe/erpnext#13110]
The way customer and supplier links are added to Non-profit Member, a link to Share Holder should also be added.

This is very important for a Cooperative Model where Member can buy, sell, have share capital in the cooperative(company). Or at least the share holder module should allow shares to be allocated to non-profit Members.(the way loan is now open for non-profit Member

Similar to 
https://github.com/frappe/erpnext/issues/12892

Share Holder Module
https://discuss.erpnext.com/t/shareholder-module-money-with-wings-moneybag-money-mouth-face/33380

**Comment by kothagunda** [comment]
Also, since share holding will be in a specific company. Is it a good idea to have a company field in the Member docType.

**Comment by kothagunda** [comment]
Hi @umairsy will this be part of erpnext Version 11

**Comment by umairsy** [comment]
Hello,

As per the updates from the meeting with Rushabh and Revant, we understand that you will be proceeding with the implementation of standard ERPNext application itself. For now, if you need more link fields, you could add it as a Custom Field as well. But it will be best to stick to the basic features as much as possible and get the initial few implementations rights.

**Comment by kothagunda** [comment]
Yeah this is true Umair, we will start with only the basic ERPNext. 
But the long term functionalities were also discussed and thus i opened these issues based on that discussion. Though we don't need it right away, i was just checking on all the issue i raised regarding the availability of the changes.
Sure we will add a custom field, if required.  Thank you for the response.

**Comment by prasannatp** [comment]
@umairsy @nabinhait can you please open this issue again.

Thank you.
